src/CameraControls.ts

```diff
@@ -116,8 +116,6 @@ export class CameraControls extends EventDispatcher {
 	mouseButtons: MouseButtons;
 	touches: Touches;
 
-	cancel: () => void = () => {};
-
```

Comment:
Could we focus only `xxxInternal()` in this PR?

Reply:
Well i thought the best was to do them all once and for all :p